src/Gamepage/Gamepage.scss

```diff
@@ -27,45 +40,52 @@
   margin-left: 15px;
   margin-top: 15px;
   margin-bottom: 15px;
-  width: 97.5vw;
-  border-radius: 1%;
-  min-height: 200px;
-  grid-row: 1;
+  width: 50vw;
+  align-self: center;
+  justify-self: center;
+  border-radius: 15px;
+  height: 15%;
+  grid-column: 2;
   align-items: center;
   @include allDivRules();
-  box-shadow: -1px 10px 20px 3px #888888;
-}
-
-.bottom {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  overflow-y: scroll;
 }
 
 .chat {
-  grid-column-start: 1;
-  grid-column-end: 1;
-  margin-left: 15px;
-  @include bottomDivRules;
-}
-
-.players {
-  grid-column-start: 2;
-  grid-column-end: 2;
-  @include bottomDivRules;
+  @include chatAndAttempts;
```

Comment:
Did I already say great job on the @mixins/@includes? Because great job. This looks awesome.